### DISCUSSION TITLE
Sort Discussions by Votes

### DISCUSSION THREAD

**Discussion by timharris777** [community/community#6541] [Discussions] [answered]
There is currently no way to sort discussions by votes. If I have an idea/feature request category. I would like to easily sort by votes so that the most upvoted idea/feature requests are at the top. Currently I have to scroll through and see which ones have more votes.

**Comment by Chaoses-Ib** [comment]
See #3328

**Comment by TPS** [comment, marked as answer]
@timharris777 Duplicate https://github.com/orgs/community/discussions/3328 is marked <kbd>Shipped</kbd>.  Close FTW?
